src/drivers/samv7/tone_alarm/tone_alarm.cpp

```diff
@@ -314,49 +230,29 @@ extern "C" __EXPORT int tone_alarm_main(int argc, char *argv[]);
 
 ToneAlarm::ToneAlarm() :
 	CDev("tone_alarm", TONEALARM0_DEVICE_PATH),
-	_default_tune_number(0),
-	_user_tune(nullptr),
-	_tune(nullptr),
-	_next(nullptr),
-	_cbrk(CBRK_UNINIT)
+	_running(false),
```

Comment:
Since px4 is now using C++11, shouldn't we initialize those members directly at definition in the class body? Would remove clutter from the constructor.

Reply:
true but not the goal at the moment.